Investigate timeouts in e2e tests running in CI

[Sometimes the e2e tests will fail to run](https://github.com/Seneca-CDOT/telescope/runs/2177747320?check_suite_focus=true):

```
> jest -c jest.config.e2e.js --


Jest dev-server output:
(Not all processes could be identified, non-owned process info
 will not be shown, you would have to be root to see it all.)
Error: Server has taken more than 5000ms to start.
    at setupJestServer (/home/runner/work/telescope/telescope/node_modules/jest-process-manager/lib/index.js:211:19)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async Promise.all (index 0)
    at async Object.setup (/home/runner/work/telescope/telescope/node_modules/jest-process-manager/lib/index.js:138:5)
    at async setup (/home/runner/work/telescope/telescope/node_modules/jest-playwright-preset/lib/global.js:25:13)
    at async /home/runner/work/telescope/telescope/node_modules/@jest/core/build/runGlobalHook.js:82:11
    at async waitForPromiseWithCleanup (/home/runner/work/telescope/telescope/node_modules/@jest/transform/build/ScriptTransformer.js:207:5)
    at async /home/runner/work/telescope/telescope/node_modules/@jest/core/build/runGlobalHook.js:72:9
    at async pEachSeries (/home/runner/work/telescope/telescope/node_modules/p-each-series/index.js:8:23)
npm ERR! code ELIFECYCLE
```

We might need to increase the 5000ms startup window so it works on the GitHub builders (I bet they are under-powered).

--- a/jest-playwright.config.js
+++ b/jest-playwright.config.js
@@ -8,7 +8,7 @@ module.exports = {
   serverOptions: {
     command: 'npx http-server src/api/auth/test/e2e -p 8888',
     port: 8888,
-    launchTime: 10000,
+    launchTimeout: 30000,
     usedPortAction: 'kill',
     debug: true,
   },

--- a/jest.config.base.js
+++ b/jest.config.base.js
@@ -6,4 +6,5 @@ module.exports = {
   coverageDirectory: '<rootDir>/coverage',
   moduleDirectories: ['node_modules'],
   automock: false,
+  testTimeout: 30000,
 };

